GitUI/HelperDialogs/FormResetCurrentBranch.cs

```diff
@@ -24,7 +24,7 @@ public enum ResetType
         }
 
         public static FormResetCurrentBranch Create(GitUICommands commands, GitRevision revision, ResetType resetType = ResetType.Mixed)
-            => new FormResetCurrentBranch(commands, revision ?? throw new NotSupportedException(TranslatedStrings.NoRevision), resetType);
+            => new(commands, revision ?? throw new NotSupportedException(TranslatedStrings.NoRevision), resetType);
```

Comment:
This omission is located at the edge of readability *for me*. Not requesting a change.

Reply:
I see your point in this case, even I prefer the change also here.
It is hard to have consistent rules without reasonable exceptions. I do not really want IDE0008 to be enforced for instance but to keep it as a recommendation where needed.